openquake/calculators/event_based_risk.py

```diff
@@ -71,10 +78,13 @@ def event_based_risk(riskinputs, riskmodel, rlzs_assoc, monitor):
         a single array of dtype elt_dt, or an empty list
     """
     lti = riskmodel.lti  # loss type -> index
-    specific_assets = monitor.oqparam.specific_assets
-    result = cube(
-        monitor.num_outputs, len(lti), len(rlzs_assoc.realizations), list)
-    for out_by_rlz in riskmodel.gen_outputs(riskinputs, rlzs_assoc, monitor):
+    L, R = len(lti), len(rlzs_assoc.realizations)
```

Comment:
Few understandable in any case.
Maybe a document with this kind of practices could help developers in the future.